analyzers/tests/SonarAnalyzer.UnitTest/TestCases/BlazorQueryParameterRoutableComponent_Noncompliant.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.AspNetCore.Components;
+using System;
+
+public class Nullable
```

Comment:
Educational: what is the purpose of this class?

Reply:
To test when the type is named Nullable as the `System.Nullable` class and add coverage 😄

Reply:
I am just seeing now that I lost some test cases